doc/subsystems/bluetooth/qualification.rst

```diff
@@ -15,3 +15,28 @@ the following documents:
    l2cap-pics.rst
    sm-pics.rst
    rfcomm-pics.rst
+
+Qualification Listings
+**********************
+
+The Zephyr BLE stack has obtained qualification listings for both the Host
+and the Controller.
+See the tables below for a list of qualification listings
+
+Host qualifications
+-------------------
+
+=============== ======================================================= ===================
```

Comment:
While this table works, you'll find using a list-table directive would be much easier to maintain:

    .. list-table::
       :header-rows: 1

       * - Zephyr version
         - Link
         - Qualifying Company

       * - 1.13
         - `Launch Studio details <https://launchstudio.bluetooth.com/ListingDetails/70189>`__
         - Nordic